daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -37,6 +37,9 @@ data Command
         , shutdownStdinClose :: Bool
         }
     | New { targetFolder :: FilePath, templateNameM :: Maybe String }
+    | CreateDamlApp { targetFolder :: FilePath }
+    -- ^ CreateDamlApp is sufficiently special that in addition to
+    -- `daml new foobar create-daml-app` we also make `daml create-daml-app foobar` work.
```

Comment:
I'm not entirely sure I agree it should get special treatment — using the generic syntax removes magic and hints at the existence of other templates — but if we do make it special, why not go a bit further? `daml create app` sounds a lot nicer to me.

Reply:
`daml create app` looks confusing. That sounds like `daml create`  also accepts other parameters. `daml create-app` would be reasonable but I’m not sure that’s better given that we refer to it as `create-daml-app` everywhere else.

As for making it a special command `daml create-daml-app foobar` seems sufficiently nicer to me for a GTG than `daml new foobar create-daml-app` that I think it makes sense to have it. (Note that `daml new foobar create-daml-app` still works if you want to use it)

Reply:
An even nicer-to-have would be `daml create-daml-app` creating a directory called `create-daml-app` by default, to avoid `daml create-daml-app create-daml-app`.

Reply:
I considered that but I’m not sure anyone wants to call their project `create-daml-app` so that seems a bit silly.

Reply:
Sort of a moot point by now, but for clarity I feel compelled to clarify that I meant it as `daml create <new-folder/project-name>`.